pkgs/development/python-modules/bitstring/default.nix

```diff
@@ -12,12 +13,18 @@ buildPythonPackage rec {
     owner = "scott-griffiths";
     repo = pname;
     rev = "bitstring-${version}";
-    sha256 = "0y2kcq58psvl038r6dhahhlhp1wjgr5zsms45wyz1naq6ri8x9qa";
+    hash = "sha256-CqeOYjZY2fA9L0RX/Ut+kocLKYQKNpPRAHTriwpmU3g=";
   };
 
-  checkInputs = [ unittestCheckHook ];
+  patches = [
+    (fetchpatch {
+      name = "fix-running-unit-tests-using-unittest-hook.patch";
+      url = "https://github.com/scott-griffiths/bitstring/pull/233/commits/5be9ddb4ff59825d8be76548695701c20e014a0d.patch";
+      hash = "sha256-+ZGywIfQQcYXJlYZBi402ONnysYm66G5zE4duJE40h8=";
+    })
+  ];
 
-  unittestFlagsArray = [ "-s" "test" ];
```

Comment:
Per the OfBorg build logs, it looks like this needs to stay, else no tests are found.

Reply:
Great catch! I will add back the `[ "-s" test" ]`. The reason, sadly, is because patches with empty `__init__.py` files appear to not create those files.